docker-compose.yaml

```diff
@@ -85,3 +85,16 @@ services:
       retries: 10
     ports:
       - "5432:5432"
+
+  jaeger:
+    image: jaegertracing/all-in-one:latest
```

Comment:
Can you add some details to docs/Contributing.md about running `docker compose --profile tracing` and how to connect to bring up jaeger? 

Also, any notes about how to update opentdf-dev.yaml?